app/mbm/static/css/custom.css

```diff
@@ -63,3 +63,61 @@ a {
   background-color: var(--dark-green-color);
   border: var(--dark-green-color);
 }
+
+.gps-location-marker-icon {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translateX(-50%) translateY(-50%);
+  width: 30px;
+  height: 30px;
+}
+
+.gps-location-marker-icon:before {
+  content: "";
+  position: relative;
+  display: block;
+  width: 300%;
+  height: 300%;
+  box-sizing: border-box;
+  margin-left: -100%;
+  margin-top: -100%;
+  border-radius: 45px;
+  background-color: #01a4e9;
+  animation: pulse-ring 1.25s cubic-bezier(0.215, 0.61, 0.355, 1) infinite;
+}
+
+.gps-location-marker-icon:after {
+  content: "";
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-color: #39d1ff83;
```

Comment:
**[Nitpick, non-blocking]** I wonder if maybe this color is a bit too muted against our background, especially at high zoom levels where it intersects with the route lines? I think a slightly darker shade would provide more contrast, e.g. the color of the marker is very legible and we may even want to copy it for consistency. What do you think?

<img width="988" alt="Screen Shot 2022-08-22 at 1 57 06 PM" src="https://user-images.githubusercontent.com/14170650/185998225-c5218b7d-817b-4d6b-9902-9feb1f9fab88.png">
<img width="1004" alt="Screen Shot 2022-08-22 at 1 59 35 PM" src="https://user-images.githubusercontent.com/14170650/185998232-16a780d1-7098-4a6c-a767-8401c18646cc.png">